src/main/java/admin/controllers/AdminGroups.java

```diff
@@ -0,0 +1,90 @@
+package admin.controllers;
+
+import javafx.event.ActionEvent;
+import javafx.event.Event;
+import javafx.event.EventType;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Alert;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import models.Group;
+
+import java.io.IOException;
+
+public class AdminGroups extends AnchorPane {
+    public static final EventType<Event> GROUPS_UPDATED =
+            new EventType<>("GROUPS_UPDATED");
+    @FXML
+    VBox entries;
+
+    public AdminGroups(){
+        initializeView();
+    }
+
+    /**
+     * Loads the JavaFX view component
+     *
+     * This can not be done in the constructor, because initialization logic of extended classes might not be done otherwise
+     */
+    protected void initializeView() {
```

Comment:
Es macht keinen Sinn das auszulagern, wenn es dann trotzdem im Constructor aufgerufen wird. Ich denke das Pattern macht eher bei den Abstrakten Basisklassen Sinn, um das dann in der abgeleiteten Klasse aufzurufen

Reply:
Stimmt, hab ich einfach von deinen AppointmentViews übernommen...